core/src/main/java/io/opencensus/stats/Stats.java

```diff
@@ -19,51 +19,57 @@
 import java.util.logging.Logger;
 import javax.annotation.Nullable;
 
-/** {@link Stats}. */
+/** Class for accessing the default {@link StatsComponent}. */
 public final class Stats {
   private static final Logger logger = Logger.getLogger(Stats.class.getName());
 
-  private static final StatsManager statsManager =
-      loadStatsManager(Provider.getCorrectClassLoader(StatsManager.class));
+  private static final StatsComponent statsComponent =
+      loadStatsComponent(Provider.getCorrectClassLoader(StatsComponent.class));
 
   /** Returns the default {@link StatsContextFactory}. */
   @Nullable
   public static StatsContextFactory getStatsContextFactory() {
-    return statsManager == null ? null : statsManager.getStatsContextFactory();
+    return statsComponent == null ? null : statsComponent.getStatsContextFactory();
```

Comment:
Instead of having all this if conditions probably easier to have a NoopStatsComponent which returns the expected value if no-op implementation.

Maybe make sense in a different PR.